dis_snek/api/http/http_client.py

```diff
@@ -154,7 +154,7 @@ def __init__(self, connector: Optional[BaseConnector] = None, loop: Optional[asy
 
     def __del__(self):
         if self.__session and not self.__session.closed:
-            self.loop.run_until_complete(self.__session.close())
+            asyncio.get_running_loop().run_until_complete(self.__session.close())
```

Comment:
Isn't there a potential that the event loop is still running at this point?

If you use `asyncio.run()` won't the event loop *have closed*?

We really should be closing the session in some other way :/.